src/test/java/com/stripe/net/LiveStripeResponseGetterTest.java

```diff
@@ -148,10 +179,12 @@ public void testAppInfo() {
 
 		Gson gson = new Gson();
 
-		Map<String, String> uaMap = gson.fromJson(headers.get("X-Stripe-Client-User-Agent"), Map.class);
+		Map<String, String> uaMap = gson.fromJson(headers.get("X-Stripe-Client-User-Agent"),
+			new TypeToken<Map<String, String>>() {}.getType());
```

Comment:
Another opportunistic warning fix: use `TypeToken` to work around the type erasure.